compressors/huffman_coder.py

```diff
@@ -85,14 +85,15 @@ class HuffmanEncoder(PrefixFreeEncoder):
     """
     PrefixFreeEncoder already has a encode_block function to encode the symbols once we define a encode_symbol function
     for the particular compressor.
-    PrefixFreeTree provides encode_symbol given a PrefixFreeTree
+    PrefixFreeTree provides get_encoding_table given a PrefixFreeTree
```

Comment:
should it be `PrefixFreeEncoder provides get_encoding_table given a PrefixFreeTree`

Reply:
yes, thanks!